components/TaggedValue/TaggedValue.jsx

```diff
@@ -0,0 +1,30 @@
+require('./TaggedValue.scss')
+
+import React from 'react'
+import classNames from 'classnames'
+
+const TaggedValue = ({title,subText,style,count}) => {  
+  const taggedValueClass = classNames(
+    'TaggedValue',
+    { [`${style}`]: style,
```

Comment:
OK I dont have issue changing to it . Actually initially it was same what you are saying
So now if in JSX tag the attribute style is not provided this is html generated(note style variable is initialised to undefined)
class = "TaggedValue undefined " or if you add default class class = "TaggedValue undefined default" 

since we have set style to true always... If required I cn go ahead and change it